src/command.js

```diff
@@ -104,7 +104,7 @@ const command = {
     if (!cfg.from) throw new Error('Missing \'from\' property in config or argument')
     const cwd = process.cwd()
     /* istanbul ignore next */
-    const serviceArgs = process.env.DEVLAB_NO_RM ? ['run', '-d', '--privileged'] : ['run', '-d', '--rm', '--privileged']
+    const serviceArgs = global.rmOnShutdown ? ['run', '-d', '--privileged'] : ['run', '-d', '--rm', '--privileged']
```

Comment:
my only concern/hestiation here is using a `global`. We have the `cfg` object, so it could be added to that.